### DISCUSSION TITLE
Logout token management

### DISCUSSION THREAD

**Discussion by Eraldo** [xtrinch/fcm-django#265] [Ideas]
I suggest also adding a method that takes care of device and subscription handling when a user logs out of the device.
Otherwise the device will keep getting notifications!

This might entail:
* Invalidating active tokens. (`active=False`)
* Unsubscribing the token from all topics.

**Comment by xtrinch** [comment]
In relation to my comment on https://github.com/xtrinch/fcm-django/discussions/264, what could be done is on device deletion, the token unsubscribed from all topics
